### PR TITLE
docs: esl-select + esl-select-list + esl-tabs vs esl-tab

### PR DESCRIPTION
**Pull request by ala-n**

### CODE REVIEW

File: src/modules/esl-forms/esl-select-list/README.md [hunk 1]
```diff
@@ -1,13 +1,37 @@
...
+HTMLSelectElement is used as model for ESLSelect.
 
-Description TBD.
+### Attributes / Properties
+
+- `select-all-label` - select all options text
+
+- `disabled` - disabled state marker
+
+- `pin-selected` - marker to top pin selected items to top in dropdown
```
Comment (dshovchko) [inline src/modules/esl-forms/esl-select-list/README.md:18]: Maybe replace by
`marker for selecting items to be pinned to the top of the dropdown`
?

File: src/modules/esl-forms/esl-select-list/core/esl-select-list.ts [hunk 2]
```diff
@@ -18,8 +24,12 @@ export class ESLSelectList extends ESLSelectWrapper {
     super.register();
   }
 
+  /** Select all options text */
   @attr({defaultValue: 'Select All'}) public selectAllLabel: string;
+
+  /** Disabled state marker */
   @boolAttr() public disabled: boolean;
+  /** Marker to top pin selected items to top in dropdown */
```
Comment (dshovchko) [inline src/modules/esl-forms/esl-select-list/core/esl-select-list.ts:32]: `marker for selecting items to be pinned to the top of the dropdown` ?

**Review by dshovchko — APPROVED** [review]

**Review by julia-murashko — APPROVED** [review]

**Review by yadamskaya — APPROVED** [review]

**Review by yadamskaya — APPROVED** [review]

**Pull request merged**